README.md

```diff
@@ -1,3 +1,33 @@
 # bevy_libgdx_atlas
 
-Support loading `libgdx` atlas files as bevy assets.
\ No newline at end of file
+[![Following released Bevy versions](https://img.shields.io/badge/Bevy%20tracking-released%20version-lightblue)](https://bevyengine.org/learn/quick-start/plugin-development/#main-branch-tracking)
+[![crates.io](https://img.shields.io/crates/v/bevy_libgdx_atlas)](https://crates.io/crates/bevy_libgdx_atlas)
+[![docs.rs](https://docs.rs/bevy_libgdx_atlas/badge.svg)](https://docs.rs/bevy_libgdx_atlas)
+
+
```

Comment:
lets list the other crates here as done in the other crates too (fine in a followup PR)

Reply:
I decided to put this in another section in the README.md since the list of crates had gotten long enough to be distracting from the actual crate the README.md is about.